dorner.py

```diff
@@ -21,7 +21,7 @@ def create_perfections() -> list[DanceEvent]:
                 ends_at=dateparser.parse("22:15"),
                 name="Perfektion",
                 price_euro_cent=700,
-                description="Freitagsperfektion I TanzZeit ",
+                description="Freitagsperfektion I TanzZeit",
```

Comment:
**chore:** I think we can remove the I in the description, even though it is on the website but I just think it kinda is misleading.

Reply:
I guess you might be right